database/dynamodb.go

```diff
@@ -61,21 +60,18 @@ func getAWSCredentials() *AWSCredentials {
 	return awsCredentials
 }
 
-func GetFeatureFlagTableName() string {
-	tableName, found := os.LookupEnv(utils.FF_TABLE_NAME)
+func GetTableName(envVarName string) string {
+	tableName, found := os.LookupEnv(envVarName)
 	if !found {
-		log.Println("Feature Flag table name is not set in env.")
-
-		// Here the error is passed for debugging to the user we show "Something went wrong
-		utils.ServerError(errors.New("Feature flag table name not stored in the env."))
+		log.Panicf("%v is not set in env", envVarName)
```

Comment:
wont the panic terminate the program.

Reply:
Updated the code to log it using `log.Printf`